Cargo.toml

```diff
@@ -73,15 +79,15 @@ zombienet-sdk = { version = "0.2.26", default-features = false }
 git2_credentials = "0.13.0"
 
 # benchmarking
-cumulus-primitives-proof-size-hostfunction = "0.10.0"
-frame-benchmarking-cli = { version = "46.0.0", default-features = false }
-sc-chain-spec = { version = "41.0.0", default-features = false }
-sp-runtime = { version = "40.0.1", default-features = false }
-sp-statement-store = "15.0.0"
+cumulus-primitives-proof-size-hostfunction = "0.12.0"
```

Comment:
These updates are needed to get the CI green, it was failing during the `pop install` step in the `arch` container. See https://github.com/r0gue-io/pop-cli/actions/runs/14766225796/job/41488228062?pr=523

While it might make more sense to open a separate PR against `main`, given the urgency to merge and release this PR, it may be quicker to introduce the fix here.